cleanup for howtos3 (#638)
openedx/docs.openedx.org: source/educators/how-tos/data/manage_course_grades.rst

   learners' grades and their experience in the course as well as analysis of
   course data.

>    Oepn edX platform recommends that you announce any unavoidable grading related changes to

Reviewer: ```suggestion
   Best practice is that you announce any unavoidable grading related changes to
```

****************************************************

For any course, you can generate grades and then download a file with the

> results for all learners in the course, including unenrolled learners. Open edX platform

Reviewer: ```suggestion
results for all learners in the course, including unenrolled learners. Best practice is to generate```

recommends that you generate a grade report as soon as certificates have been
issued for your course.

When you initiate calculations to grade learner work, a process starts on the

> Open edX servers. The complexity of your grading configuration and the number of

Reviewer: ```suggestion
your instance's servers. The complexity of your grading configuration and the number of
```

  specific graded subsection. For information about how to do this, see
  :ref:`Override in Gradebook`.

> * **Master’s courses:** The Open edX gradebook has several features for high-touch courses. For learners in the Master’s track:

Reviewer: I think we can remove everything about master's courses as that's edx.org-specific, but I'd check in with @jswope00 about that.

Reviewer: @jesgreco please remove all content about Master's courses

Reviewer 2: agree that references to "Master's courses" should be removed.

.. _Override Subsection Bulk:

******************************************
Override Learner Subsection Scores in Bulk
******************************************

> .. note:: Master’s-only feature

Reviewer: Same here. I think Master's only is edx.org-specific. @jswope00 ?

   The CSV contains one row per learner, and the following columns:

>    * **username**: The user's Open edX username

Reviewer: ```suggestion
   * **username**: The user's username
```

The report file contains per-learner information for learners in the Master’s track, including:

> * Open edX user ID

Reviewer: ```suggestion
* user ID
```

> * Open edX username

Reviewer: ```suggestion
* username
```

You can also use this tab to **view all deadlines** or **reset a deadline**.

> *This feature is supported in courses published after June 1, 2019. To enable for an older course, first Publish the course from Studio. If you have questions, please contact your Open edX Project Coordinator.*

Reviewer: ```suggestion
*This feature is supported in courses published after June 1, 2019. To enable for an older course, first Publish the course from Studio.*
```

Find the Unique Location Identifier for a Problem
==================================================

> When you create each of the problems for a course, Open edX assigns a unique

Reviewer: ```suggestion
When you create each of the problems for a course, the platform assigns a unique
```
